doc/release/upcoming_changes/15006.new_feature.rst

```diff
@@ -0,0 +1,7 @@
+crackfortran has support for operator and assignement overloading
+-----------------------------------------------------------------
+``crackfortran`` parser is now understanding operator and assignement
```

Comment:
'is now understanding' --> 'now understands'

Reply:
Corrected.